src/App.vue

```diff
@@ -59,9 +59,6 @@ export default {
       this.reloadData();
     });
     setInterval(() => this.reloadData(), 120 * 1000);
-    EventBus.$on('backendError', () => this.$route.name !== 'maintenance' && this.$router.push({
-      name: 'maintenance',
-    }).catch((err) => { console.error(err); }));
```

Comment:
Seems that you have restored a different version of App.vue 🤷‍♀️

Reply:
OK, revert to the current one from the `develop` branch